spec/services/fees/one_off_service_spec.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Fees::OneOffService do
+  subject(:one_off_service) do
+    described_class.new(invoice:, fees:)
+  end
+
+  let(:invoice) { create(:invoice, organization:) }
+  let(:organization) { create(:organization) }
+  let(:add_on_first) { create(:add_on, organization:) }
+  let(:add_on_second) { create(:add_on, amount_cents: 400, organization:) }
+  let(:fees) do
+    [
+      {
+        add_on_code: add_on_first.code,
+        unit_amount_cents: 1200,
+        units: 2,
+        description: 'desc-123'
```

Comment:
Style/TrailingCommaInHashLiteral: Put a comma after the last item of a multiline hash.